revprxy.yaml

```diff
@@ -0,0 +1,160 @@
+- hosts: 34.121.46.70
+  gather_facts: no
+  remote_user: ubuntu
+  become: true
+
+
+  tasks:
+    - name: ensure nginx is at the latest version
+      apt:
+        name=nginx
+        state=latest
+        update_cache=yes
+
+    - name: start nginx
+      service:
+        name: nginx
+        state: started
+
+    - name: create CA key
+      openssl_privatekey:
+        path: /root/CA_key.pem
+      register: ca_key
+
+    - name: create the CA CSR
+      openssl_csr:
+        path: /root/CA.csr
+        privatekey_path: "{{ ca_key.filename }}"
+        common_name: 34.121.46.70
+      register: ca_csr
+
+    - name: sign the CA CSR
+      openssl_certificate:
+        path: /root/CA.crt
+        csr_path: "{{ ca_csr.filename }}"
+        privatekey_path: "{{ ca_key.filename }}"
+        provider: selfsigned
+      register: ca_crt
+
+    - name: create host CSR signing key
+      openssl_privatekey:
+        path: /root/example_com_host_key.pem
+      register: example_com_key
+
+    - name: create the CSR for Nginx host
+      openssl_csr:
+        path: /root/example_com.csr
+        privatekey_path: "{{ example_com_key.filename }}"
+        common_name: 34.121.46.70
```

Comment:
This should not be hardcoded, should be used as variable

Reply:
Used variables as suggested.